src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java

```diff
@@ -860,7 +862,7 @@ public Object createValue(UIDefaults defaultsTable) {
             //"TabbedPane.shadow", table.get("controlShadow"),
             //"TabbedPane.darkShadow", table.get("controlDkShadow"),
             //"TabbedPane.focus", table.get("controlText"),
-            "TabbedPane.opaque", useOpaqueComponents,
+            "TabbedPane.opaque", false,
```

Comment:
Looks like `useOpaqueComponents` is normally set to `Boolean.TRUE`. But now we set it to `false`. Why not `Boolean.FALSE`?